tests/smoke-tests/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "module",
+  "name": "@sap-ai-sdk/smoke-tests",
+  "version": "0.0.0",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "private": "true",
+  "scripts": {
+    "start": "node dist/server.js",
+    "add-canary": "pnpm add @sap-ai-sdk/ai-core@canary --save-prod=false --save-dev=false && pnpm add @sap-ai-sdk/gen-ai-hub@canary --save-prod=false --save-dev=false",
+    "create-deployment": "pnpm add-canary && rm -rf dist && cp ../../sample-code/dist dist",
+    "cf-login": "dotenv -- bash -c 'cf login -a \"$CF_API_URL\" -u \"$CF_USER\" -p \"$CF_PASSWORD\" -o \"$CF_ORG\" -s \"$CF_SPACE\"'",
```

Comment:
Why dotenv?

Reply:
It sets the env variables from the .env, so you can reference them in the command after this. Of course you can login manually, but I found this much more convenient